db/models.py

```diff
@@ -46,3 +54,66 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE)
```

Comment:
bad formmating(

Reply:
![image](https://github.com/mate-academy/py-tickets-and-orders/assets/84961728/c015605a-ef0c-4173-bc39-f4820499902e)
![image](https://github.com/mate-academy/py-tickets-and-orders/assets/84961728/316f3bd6-c97a-487f-ad1d-da24eed9c8fb)